test-suite/src/test/scala/org/scalajs/testsuite/javalib/math/BigDecimalArithmeticTest.scala

```diff
@@ -1,3 +1,4 @@
+// scalastyle:off
```

Comment:
Quite a few errors. I didn't actually investigate, since it was 1) the tests and 2) ported from Java. Maybe one day I'll revisit this.